src/main/java/core/basesyntax/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void operateTransaction(FruitTransaction transaction) {
+        if (transaction.getQuantity() < 0) {
+            throw new RuntimeException("Balance should be positive.");
+        }
+        Storage storage = new Storage();
```

Comment:
no need to create instance of storage here. make put method static or use map directly. fix in all handlers